src/main/java/com/hackclub/hccore/playerMessages/afk/RunAFKMessage.java

```diff
@@ -7,7 +7,7 @@
 public class RunAFKMessage {
 
   final static String minimsgSource = """
-      Run /afk to mark yourself as active""";
+      Run /notafk to mark yourself as active""";
```

Comment:
Was the actual command updated to reflect this?
Here, afk is used as a toggle on and off

Reply:
i dont know where the code for the command is
i found it odd that when you want to be active you run /afk and wanted to change it to /notafk